src/plugins/orangehrmCorePlugin/Authorization/Manager/BasicUserRoleManager.php

```diff
@@ -917,4 +917,37 @@ protected function fixUserRoleNameForWorkflowStateMachine(string $roleName, stri
 
         return $fixedName;
     }
+
+    /**
+     * Returns an array of accessible quick launch items
+     * e.g. - ['leave.leave_list', 'leave.my_leave', 'time.my_timesheet']
+     *
+     * @param string[] $rolesToExclude
+     * @param string[] $rolesToInclude
+     * @param array $requiredPermissions
+     * @return string[]
+     */
+    public function getAccessibleQuickLaunchList(
```

Comment:
define an abstract method in the parent. `AbstractUserRoleManager`.
This will lead to breaking unit tests. have to update them also